### PR TITLE
Surveys intro

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
This adds the current design for permissions onboarding to the iOS UI. I decided to use that as a first "survey" b/c it was something we know we will need for September and it allowed me to test out the UI/UX of a current design. I'm still new at SwiftUI and I'm not sure I have all the bindings correct, but it was time to submit something b/c it's getting big.

Note: some code is included that stubs out text entry b/c I started working on that before discovering that we never fixed the issue with Kotlin generics in obj-c code and that's going to be a more complicated fix.

### CODE REVIEW

File: iosApp/iosApp/AppDelegate.swift [hunk 1]
```diff
@@ -1,80 +1,27 @@
...
-// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
-// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
-// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
-// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
-// OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-//
-
-
-import SwiftUI
```
Comment (Erin-Mounts) [inline iosApp/iosApp/AppDelegate.swift:35]: Looks like this file got re-created/replaced with a different default template? In any case, could use a license comment block at the top

File: iosApp/iosApp/Bundle+module.swift [hunk 2]
```diff
@@ -31,17 +31,16 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //
 
+import Foundation
 
-import SwiftUI
+// Swift Packages have an internal static property defined on the Bundle to access
+// bundle resources. This code file is *not* included in the swift packages and can
+// allow building both a dynamic framework *or* a Swift Package using the same code
```
Comment (Erin-Mounts) [inline iosApp/iosApp/Bundle+module.swift:38]:
```suggestion
// allow building either a dynamic framework *or* a Swift Package using the same code
```

File: iosApp/iosApp/SceneDelegate.swift [hunk 3]
```diff
@@ -1,52 +1,33 @@
...
-// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
-// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
-// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
-// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
-// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
-// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
-// OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-//
-
```
Comment (Erin-Mounts) [inline iosApp/iosApp/SceneDelegate.swift:33]: Did you mean to delete the license comment block? Or is this another case of an autogenerated file replacing one with the same name

**Review by Erin-Mounts — APPROVED** [review]
Couple of nits/questions, otherwise looks good

**Review by nategbrown9 — APPROVED** [review]
Looks good.

**Pull request merged**